Student_BCOs/Evan_Littleton_BCO_000546

```diff
@@ -0,0 +1,193 @@
+    ],
+    "extension_domain": {},
+    "description_domain": {
+        "keywords": [
+            "STAT3",
+            "STAT3 knockout",
+            "Multi-omic genome-wide analysis",
+            "CRISPR-Cas9",
+            "Ovarian cancer"
+        ],
+        "pipeline_steps": [
+            {
+                "name": "STAR",
+                "version": "2.5.2",
+                "step_number": 1,
+                "input_list": [
+                    {
+                        "access_time": "2019-10-15 T15:20:45",
+                        "uri": "https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134375",
+                        "filename": "SRA files"
+                    }
+                ],
+                "output_list": [
+                    {
+                        "access_time": "",
+                        "uri": "",
+                        "filename": "Publication did not give output/input files for any steps other than inputs for step 1 and outputs from step 4"
+                    }
+                ],
+                "prerequisite": [],
+                "description": "Alignment of reads to reference genome"
+            },
+            {
+                "name": "Cufflinks",
+                "version": "2.2.1",
+                "step_number": 2,
+                "input_list": [],
+                "output_list": [],
+                "prerequisite": [],
+                "description": "Quantification of aligned reads"
+            },
+            {
+                "name": "featureCounts",
+                "version": "1.6.1",
+                "step_number": 3,
+                "input_list": [],
+                "output_list": [],
+                "prerequisite": [],
+                "description": "Attainment of gene read counts from quantification"
+            },
+            {
+                "name": "DESeq2",
+                "version": "1.18.1",
+                "step_number": 4,
+                "input_list": [],
+                "output_list": [
+                    {
+                        "access_time": "2019-10-15 T15:25:30",
+                        "uri": "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6735387/bin/thnov09p5478s1.pdf",
+                        "filename": "Supplemental Data"
+                    }
+                ],
+                "prerequisite": [],
+                "description": "Differential expression analysis of RNA-seq data"
+            }
+        ]
+    },
+    "execution_domain": {
+        "external_data_endpoints": [
+            {
+                "name": "Publication gave no information regarding scripts, parameters, or where the pipeline was executed",
```

Comment:
Instead of placing this text here, I would put this into the usability domain as "No information regarding  x, y, z...was found in the publication"